containers/java-postgres/README.md

```diff
@@ -0,0 +1,157 @@
+# Java & PostgreSQL
+
+- [Java & PostgreSQL](#java--postgresql)
```

Comment:
The metadata is parsed, so this index needs to go beneath it.

Reply:
Ah, thank you. I can do that!